validator/db/kv/attestation_history.go

```diff
@@ -44,16 +61,18 @@ func (store *Store) AttestationHistoryForPubKeys(
 		copy(ehd, ah)
 		attestationHistoryForVals[pk] = ehd
 	}
-	return attestationHistoryForVals, err
+
+	return attestationHistoryForVals, minAttForVal, err
```

Comment:
Agree with these changes, we just need to make sure to amend the godoc for the db function to clarify what it returns